src/app/client/src/assets/styles/themes.scss

```diff
@@ -1194,4 +1194,77 @@ html[layout='joy'] {
     width: calc(100% - 4%);
     margin: 0 auto;
   }
+}
+
+html[layout='joy'] {
+  --df-primary-bg: #FFD954;
+  --df-gray-bg: #E9E8D9;
+  --sbt-bradius: 0.125rem;
+  --sbt-bradius-xs: calc(var(--sbt-bradius) * 4);  /* 8 value */
+  --sbt-bradius-sm: calc(var(--sbt-bradius) * 8);  /* 16 value */
+  --sbt-bradius-md: calc(var(--sbt-bradius) * 12);  /* 24 value */
+  --sbt-bradius-lg: calc(var(--sbt-bradius) * 16);  /* 32 value */
```

Comment:
Till here the code is right. Below there are too many classes are added. We shouldn't add any classes here.